Add "OR" Button to Mobile Jam Control, Add Timeout Editor Page to Mobile Controls

official-sounding opened the pull request:
Added OR Buttons to the Mobile Jam Control.  Also added a new view to the Mobile Controls that allows the mobile user to edit the number of timeouts and official reviews each team has, without having to open the operator screen.  

This view is very rudimentary, mostly because jquery-ui grid layouts remain a profound mystery to me.  Someone more layout-oriented could probably make this look a bit better.  But for now i think it will work.

Closes #41

official-sounding committed:
add ability to trigger OR from jam control page

official-sounding committed:
add (extremely) rudimentary timeout editing page

allow mobile jam timer to edit timeouts and official reviews.  needs a major redo to make the UI actually look good, and add "Retained" buttons where necessary.  But gets the job done

brian-brazil commented on html/controls/mobile.js:
```diff
@@ -15,7 +15,8 @@ $sb(function() {
 	setupPeriodTimePage();
 	setupTeamScorePage();
-	setupPenaltyTimePage();
+	//setupPenaltyTimePage();
```
Why is this commented out?

official-sounding replied:
the UI side of the penalty Timing page is already commented out, wanted to prevent any of the js side of initializing as well.  I'm happy to remove the line instead of commenting out if you'd prefer.

brian-brazil replied:
I'd prefer it removed, we can always get it back from source control.

brian-brazil commented on html/controls/mobile.html:
```diff
@@ -125,20 +125,41 @@ <h1>Jam Control</h1>
 				<span class="Team1 AlternateName"></span>
+				
 			</button></div>
 			<div class="ui-block-b"><button class="Official">Official</button></div>
```
Can it be made clearer this is an OTO rather than a OR?

official-sounding committed:
fixes from code review

- make it clearer that "Official" button refers to an OTO instead of an OR
- remove, rather than comment out, ui & js call to penalty timing page in mobile screen (keep the html & js code around for now though)

brian-brazil commented on html/controls/mobile.html:
```diff
@@ -125,20 +122,41 @@ <h1>Jam Control</h1>
+				
 			</button></div>
-			<div class="ui-block-b"><button class="Official">Official</button></div>
+			<div class="ui-block-b"><button class="Official">TO Official</button></div>
```
I've already heard this as "official time out" rather than "time out official". Is this one of those things that varies across leagues?

official-sounding replied:
Official Timeout is definitely the preferred nomenclature, but using "TO Official" was more about keeping the buttons consistent with "TO Team 1" and "TO Team 2".  Also when the buttons are small, the text is truncated, so I wanted to make it clearer it was a TO instead of an OR (so it shows up as "TO Off..." instead of "Offici..." or the like)

brian-brazil commented:
Thanks!

brian-brazil commented on html/controls/mobile.html:
```diff
@@ -150,17 +168,61 @@ <h1>Period Time</h1>
 	</div>
 </div>
 
+<div data-role="page" id="TeamTimeoutPage">
```
Just testing this now, and I noticed that this doesn't appear on the mobile control index.
